src/extension.ts

```diff
@@ -75,6 +76,16 @@ export async function activate(context: vscode.ExtensionContext) {
 		console.error(`Error during post-update actions: ${errorMessage}, Stack trace: ${error.stack}`)
 	}
 
+	// backup id in case vscMachineID doesn't work
+	let installId = context.globalState.get<string>("installId")
+
+	if (!installId) {
+		installId = uuidv4()
```

Comment:
why are we using uuid instead of vscmachineid?

Reply:
It's a backup for when machineId is not available